pkgs/by-name/ta/tacent/package.nix

```
@@ -0,0 +1,37 @@
+{
+  cmake,
+  fetchFromGitHub,
+  lib,
+  ninja,
+  stdenv,
+}:
+
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```
I think that the `finalAttrs` pattern is now recommended.